src/main/kotlin/no/nav/tilleggsstonader/sak/vedtak/barnetilsyn/dto/VedtakTilsynBarnDto.kt

```diff
@@ -20,6 +20,7 @@ import no.nav.tilleggsstonader.sak.vedtak.dto.VedtakResponse
 )
 sealed class VedtakTilsynBarnDto(
     open val type: TypeVedtak,
+    val vedtaksperioder: List<VedtaksperiodeTilsynBarnDto>? = null,
```

Comment:
Denne gjelder ikke for opphør eller avslag

Reply:
Sant, trenger kanksje ikke ha den med her nei. Jeg fjerner den